battDB/forms.py

```diff
@@ -320,6 +325,10 @@ class NewExperimentDataFileForm(DataCreateForm):
     Add a new experiment data file.
     """
 
+    devices = forms.ModelMultipleChoiceField(
+        queryset=Device.objects.get_queryset(),
+    )
```

Comment:
This is empty unless you add a devices batch to the experiment, which means you then can't submit the form because it is a mandatory field. If this is intentional, then a little bit on info text explaining that devices need to be added to the experiment would be helpful

Reply:
Yes, similar to above this is intentional but you are right - some warning text on the form to say something along the lines of "you can't add any experimental data until you add devices to the experiment. Go back and do this first." would be extremely helpful!

Reply:
Added a small box with instructions if there are no devices. Red/Danger seems most appropriate as the form cannot be submitted without rectifying this.

<img width="570" alt="image" src="https://user-images.githubusercontent.com/5871253/208422915-886346f7-50b4-4662-9967-3ab59b45cb96.png">